test/visual/stories.js

```diff
@@ -11,6 +11,8 @@ governing permissions and limitations under the License.
 */
 module.exports = [
     'action-menu--default',
+    'action-menu--customIcon',
+    'action-menu--iconOnly',
```

Comment:
Can you help make sure whether the instructions to fill out this list with the "export name" vs the "story path" for this were clear? This needs to be the value seen in the URL for this story:
 
![image](https://user-images.githubusercontent.com/1156657/91626464-3d7ce300-e97d-11ea-9b8f-737fcbf4d799.png)

Storybook does some magic conversions here, and we could do it at some point, but it's not available currently. What I'd really like to get is to a place where we leverage the listing the Storybook makes on it's own or a more test runner centric version fo the same rather than this manual array.

Similarly, a nicer error/fail message than the following is probably in order here:
![image](https://user-images.githubusercontent.com/1156657/91626513-b9772b00-e97d-11ea-8446-80f2867d93cd.png)

Not withstanding, update these to strings to `action-menu--custom-icon` and `action-menu--icon-only` and your tests will run appropriately.

Reply:
I think the only thing that would have helped more would be a more obvious way to test these without running the full visual regression suite.

I made an untested "hopefully this works" change, and clearly didn't look hard enough at the format. I figured the CI would catch issues (and actually noticed these were missing last night so was pretty sure that'd be it).